package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@datapunt/matomo-tracker-js",
-  "version": "0.0.2",
+  "version": "0.0.3",
+  "private": true,
   "description": "Matomo tracker for frontend projects",
```

Comment:
name, description, main, license, author, homepage and keywords can be all removed / moved from this root package.json to the packages package.json's